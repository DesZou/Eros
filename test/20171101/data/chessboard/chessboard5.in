6
oooooo
oooooo
oooooo
ooooo*
oooooo
oooooo
