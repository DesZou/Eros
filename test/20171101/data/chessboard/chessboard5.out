1
585322013
60496891
711005184
2912032
2016
0
